Description: Update references to home location.
 .
 ~/.config/breezy rather than ~/.bazaar.
Origin: commit, revision id: jelmer@jelmer.uk-20190215174117-o9w1am2z88mg9g1q
Author: Jelmer VernooÄ³ <jelmer@jelmer.uk>
Last-Update: 2019-02-15
Applied-Upstream: no
X-Bzr-Revision-Id: jelmer@jelmer.uk-20190215174117-o9w1am2z88mg9g1q

=== modified file 'breezy/builtins.py'
--- old/breezy/builtins.py	2019-02-09 03:23:20 +0000
+++ new/breezy/builtins.py	2019-02-15 17:41:17 +0000
@@ -348,7 +348,7 @@
         Not versioned and not matching an ignore pattern.
 
     Additionally for directories, symlinks and files with a changed
-    executable bit, Bazaar indicates their type using a trailing
+    executable bit, Breezy indicates their type using a trailing
     character: '/', '@' or '*' respectively. These decorations can be
     disabled using the '--no-classify' option.
 
@@ -1728,7 +1728,7 @@
     If the tree's branch is bound to a master branch, brz will also update
     the branch from the master.
 
-    You cannot update just a single file or directory, because each Bazaar
+    You cannot update just a single file or directory, because each Breezy
     working tree has just a single basis revision.  If you want to restore a
     file that has been removed locally, use 'brz revert' instead of 'brz
     update'.  If you want to restore a file to its state in a previous
@@ -1871,11 +1871,11 @@
 class cmd_remove(Command):
     __doc__ = """Remove files or directories.
 
-    This makes Bazaar stop tracking changes to the specified files. Bazaar will
+    This makes Breezy stop tracking changes to the specified files. Breezy will
     delete them if they can easily be recovered using revert otherwise they
     will be backed up (adding an extension of the form .~#~). If no options or
-    parameters are given Bazaar will scan for files that are being tracked by
-    Bazaar but missing in your tree and stop tracking them for you.
+    parameters are given Breezy will scan for files that are being tracked by
+    Breezy but missing in your tree and stop tracking them for you.
     """
     takes_args = ['file*']
     takes_options = ['verbose',
@@ -3091,9 +3091,10 @@
     using this command or directly by using an editor, be sure to commit
     it.
 
-    Bazaar also supports a global ignore file ~/.bazaar/ignore. On Windows
-    the global ignore file can be found in the application data directory as
-    C:\\Documents and Settings\\<user>\\Application Data\\Bazaar\\2.0\\ignore.
+    Breezy also supports a global ignore file ~/.config/breezy/ignore. On
+    Windows the global ignore file can be found in the application data
+    directory as
+    C:\\Documents and Settings\\<user>\\Application Data\\Breezy\\3.0\\ignore.
     Global ignores are not touched by this command. The global ignore file
     can be edited directly using an editor.
 
@@ -3732,7 +3733,7 @@
     unreferenced ancestors
         Texts that are ancestors of other texts, but
         are not properly referenced by the revision ancestry.  This is a
-        subtle problem that Bazaar can work around.
+        subtle problem that Breezy can work around.
 
     unique file texts
         This is the total number of unique file contents
@@ -3744,7 +3745,7 @@
         entries are modified, but the file contents are not.  It does not
         indicate a problem.
 
-    If no restrictions are specified, all Bazaar data that is found at the given
+    If no restrictions are specified, all data that is found at the given
     location will be checked.
 
     :Examples:
@@ -3786,10 +3787,10 @@
     __doc__ = """Upgrade a repository, branch or working tree to a newer format.
 
     When the default format has changed after a major new release of
-    Bazaar, you may be informed during certain operations that you
+    Bazaar/Breezy, you may be informed during certain operations that you
     should upgrade. Upgrading to a newer format may improve performance
     or make new features available. It may however limit interoperability
-    with older repositories or with older versions of Bazaar.
+    with older repositories or with older versions of Bazaar or Breezy.
 
     If you wish to upgrade to a particular format rather than the
     current default, that can be specified using the --format option.
@@ -3811,7 +3812,7 @@
     If the conversion of a branch fails, remaining branches are still
     tried.
 
-    For more information on upgrades, see the Bazaar Upgrade Guide,
+    For more information on upgrades, see the Breezy Upgrade Guide,
     https://www.breezy-vcs.org/doc/en/upgrade-guide/.
     """
 
@@ -4312,7 +4313,7 @@
     through OTHER, excluding BASE but including OTHER, will be merged.  If this
     causes some revisions to be skipped, i.e. if the destination branch does
     not already contain revision BASE, such a merge is commonly referred to as
-    a "cherrypick". Unlike a normal merge, Bazaar does not currently track
+    a "cherrypick". Unlike a normal merge, Breezy does not currently track
     cherrypicks. The changes look like a normal commit, and the history of the
     changes from the other branch is not stored in the commit.
 
@@ -5133,8 +5134,8 @@
 
     --verbose shows the path where each plugin is located.
 
-    A plugin is an external component for Bazaar that extends the
-    revision control system, by adding or replacing code in Bazaar.
+    A plugin is an external component for Breezy that extends the
+    revision control system, by adding or replacing code in Breezy.
     Plugins can do a variety of things, including overriding commands,
     adding new commands, providing additional network transports and
     customizing log output.
@@ -5498,7 +5499,7 @@
     :Examples:
         brz break-lock
         brz break-lock brz+ssh://example.com/brz/foo
-        brz break-lock --conf ~/.bazaar
+        brz break-lock --conf ~/.config/breezy
     """
 
     takes_args = ['location?']
@@ -5789,7 +5790,7 @@
     branch containing submit_branch in its ancestory without needing access to
     the source branch.
 
-    If --no-bundle is specified, then Bazaar doesn't send the contents of the
+    If --no-bundle is specified, then Breezy doesn't send the contents of the
     revisions, but only a structured request to merge from the
     public_location.  In that case the public_branch is needed and it must be
     up-to-date and accessible to the recipient.  The public_branch is always

=== modified file 'breezy/globbing.py'
--- old/breezy/globbing.py	2018-11-18 19:48:57 +0000
+++ new/breezy/globbing.py	2019-02-15 17:41:17 +0000
@@ -248,7 +248,9 @@
             # the combined pattern we sent to regex. Instead we indicate to
             # the user that an ignore file needs fixing.
             mutter('Invalid pattern found in regex: %s.', e.msg)
-            e.msg = "File ~/.bazaar/ignore or .bzrignore contains error(s)."
+            e.msg = (
+                "File ~/.config/breezy/ignore or "
+                ".bzrignore contains error(s).")
             bad_patterns = ''
             for _, patterns in self._regex_patterns:
                 for p in patterns:

=== modified file 'breezy/ignores.py'
--- old/breezy/ignores.py	2018-11-12 01:41:38 +0000
+++ new/breezy/ignores.py	2019-02-15 17:41:17 +0000
@@ -33,7 +33,7 @@
     )
 """)
 
-# ~/.bazaar/ignore will be filled out using
+# ~/.config/breezy/ignore will be filled out using
 # this ignore list, if it does not exist
 # please keep these sorted (in C locale order) to aid merging
 USER_DEFAULTS = [

=== modified file 'doc/developers/apport.txt'
--- old/doc/developers/apport.txt	2009-12-02 20:34:07 +0000
+++ new/doc/developers/apport.txt	2019-02-15 17:41:17 +0000
@@ -41,7 +41,7 @@
 
 #. An exception reaches the top-level handler.
 
-#. We log it in apport-format to a file in ~/.bazaar/crash.
+#. We log it in apport-format to a file in ~/.config/breezy/crash.
 
 #. We tell the user where that file is, and invite them to file a bug
    report.

=== modified file 'doc/developers/contribution-quickstart.txt'
--- old/doc/developers/contribution-quickstart.txt	2019-01-01 21:38:07 +0000
+++ new/doc/developers/contribution-quickstart.txt	2019-02-15 17:41:17 +0000
@@ -46,7 +46,7 @@
 closely-related cluster of bugs per branch, to make reviews and merges
 flow more smoothly.
 
-You probably want this configuration in ``~/.bazaar/locations.conf``::
+You probably want this configuration in ``~/.config/breezy/locations.conf``::
 
     [/home/USER/brz]
     push_location = lp:~LAUNCHPAD_USER/brz/{branchname}

=== modified file 'doc/developers/plugin-development.txt'
--- old/doc/developers/plugin-development.txt	2018-11-22 02:23:26 +0000
+++ new/doc/developers/plugin-development.txt	2019-02-15 17:41:17 +0000
@@ -31,7 +31,7 @@
 To ensure your plugin under development is available to Breezy, set
 the ``BRZ_PLUGIN_PATH`` environment variable to its parent directory.
 Alternatively, you may wish to develop your plugin within a directory
-under your personal plugins area (``~/.bazaar/plugins`` on GNU/Linux)
+under your personal plugins area (``~/.config/breezy/plugins`` on GNU/Linux)
 or put a symbolic link in that area pointing to your plugin under
 test.  Finally you can use ``BRZ_PLUGINS_AT`` to point to a specific
 directory for a specific plugin (separated by your platform's value of

=== modified file 'doc/en/admin-guide/hooks-plugins.txt'
--- old/doc/en/admin-guide/hooks-plugins.txt	2018-11-18 13:20:01 +0000
+++ new/doc/en/admin-guide/hooks-plugins.txt	2019-02-15 17:41:17 +0000
@@ -25,10 +25,10 @@
 http://doc.bazaar.canonical.com/plugins/en/.  For purposes of installation,
 plugins are simply python packages.  They can be installed alongside Bazaar in
 the ``bzrlib.plugins`` package using each plugin's ``setup.py``.  They can
-also be installed in the plugin path which is the user's ``~/.bazaar/plugins``
-directory or can be specified with the ``BZR_PLUGIN_PATH`` environment
-variable.  See ``bzr help configuration`` for more on specifying the location
-of plugins.
+also be installed in the plugin path which is the user's
+``~/.config/breezy/plugins`` directory or can be specified with the
+``BZR_PLUGIN_PATH`` environment variable.  See ``bzr help configuration`` for
+more on specifying the location of plugins.
 
 
 Email Notification

=== modified file 'doc/en/admin-guide/migration.txt'
--- old/doc/en/admin-guide/migration.txt	2018-11-18 13:20:01 +0000
+++ new/doc/en/admin-guide/migration.txt	2019-02-15 17:41:17 +0000
@@ -56,7 +56,7 @@
 This command has flexible ways to specify what paths within the Subversion
 repository contain branches and which contain tags.  For example, the
 recommended layout for Subversion projects (called ``trunk`` by the svn
-plugin) could be specified in ``~/.bazaar/subversion.conf`` as
+plugin) could be specified in ``~/.config/breezy/subversion.conf`` as
 
 ::
 

=== modified file 'doc/en/user-guide/controlling_registration.txt'
--- old/doc/en/user-guide/controlling_registration.txt	2017-05-21 13:41:54 +0000
+++ new/doc/en/user-guide/controlling_registration.txt	2019-02-15 17:41:17 +0000
@@ -95,12 +95,12 @@
 There are some ignored files which are not project specific, but more user
 specific. Things like editor temporary files, or personal temporary files.
 Rather than add these ignores to every project, brz supports a global
-ignore file in ``~/.bazaar/ignore`` [#]_. It has the same syntax as the
+ignore file in ``~/.config/breezy/ignore`` [#]_. It has the same syntax as the
 per-project ignore file.
 
 .. [#] On Windows, the users configuration files can be found in the
-   application data directory. So instead of ``~/.bazaar/branch.conf``
+   application data directory. So instead of ``~/.config/breezy/branch.conf``
    the configuration file can be found as:
-   ``C:\Documents and Settings\<username>\Application Data\Breezy\2.0\branch.conf``.
+   ``C:\Documents and Settings\<username>\Application Data\Breezy\3.0\branch.conf``.
    The same is true for ``locations.conf``, ``ignore``, and the
    ``plugins`` directory.

=== modified file 'doc/en/user-guide/plugins.txt'
--- old/doc/en/user-guide/plugins.txt	2017-05-21 13:41:54 +0000
+++ new/doc/en/user-guide/plugins.txt	2019-02-15 17:41:17 +0000
@@ -35,7 +35,7 @@
 
 Installing a plugin is very easy! If not already created, create a
 ``plugins`` directory under your Breezy configuration directory,
-``~/.bazaar/`` on Unix and
+``~/.config/breezy/`` on Unix and
 ``C:\Documents and Settings\<username>\Application Data\Breezy\2.0\``
 on Windows. Within this directory (referred to as $BZR_HOME below),
 each plugin is placed in its own subdirectory.
@@ -45,7 +45,7 @@
 one can perform the following::
 
     brz branch http://panoramicfeedback.com/opensource/brz/brztools
-    ~/.bazaar/plugins/brztools
+    ~/.config/breezy/plugins/brztools
 
 When installing plugins, the directories that you install them in must
 be valid python identifiers. This means that they can only contain

